Makefile

```diff
@@ -1,9 +1,12 @@
 .PHONY: all clean
-.PHONY: apiserver ui
+.PHONY: apiserver ui dev
```

Comment:
My OCD would prefer if `dev` was on the same line as `all` and `clean`. Its a "utility" target like those.